awscli/customizations/waiters.py

```diff
@@ -181,6 +183,13 @@ def _build_operation_description(self, operation):
         operation_name = xform_name(operation).replace('_', '-')
         return u'when polling with ``%s``.' % operation_name
 
+    def _build_polling_description(self, delay, max_attempts):
+        description = (
+            ' It will poll every %s seconds until a successful state '
+            'has been reached. An error is thrown after %s failed checks.'
```

Comment:
Can we reword this to be more cli-centric.  This will exit with an RC of <some rc> after xx many failed checks.  The CLI doesn't really throw exceptions.